src/adatper.locale/vue-i18n.ts

```diff
@@ -0,0 +1,14 @@
+import type { I18n } from "vue-i18n";
```

Comment:
if user will not have an `vue-i18n` package it invokes error.

Please remove this type and use somthing more general. For our library case -- it's ok.

Reply:
That's i18n adapter, and it's supposed to be used with i18n only.

Reply:
reasonable, ok